data/mods/Magiclysm/items/recipe_books.json

```diff
@@ -56,5 +56,25 @@
     "intelligence": 9,
     "time": "20 m",
     "fun": -2
+  },
+  {
+    "id": "techno_basic",
+    "type": "BOOK",
+    "name": { "str": "Magitek Illustrated", "str_pl": "copies of Magitek Illustrated" },
+    "description": "A paperback tome for the art of combining modern technology with magic.  There is a disclaimer for mechanically unusual designs on the back.",
```

Comment:
disclaimer? maybe you were thinking advertisement?

Reply:
Need to reword that, disclaimer was meant to refer to the shotcestus not being fire-able if you lose the strength requirement from pain